src/extension/app/aem-sidekick.js

```diff
@@ -68,6 +68,8 @@ export class AEMSidekick extends MobxLitElement {
       >
         <main>
           <action-bar></action-bar>
+          <toast-container></toast-container>
+          <modal-container></modal-container>
```

Comment:
In the future this may get grouped into another component or included in the `action-bar` if we add the ability to drag the action-bar around the page.